src/canvas/SelectableCanvas.ts

```diff
@@ -762,30 +750,26 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
 
   /**
    * Checks point is inside the object.
-   * @param {Object} [pointer] x,y object of point coordinates we want to check.
    * @param {FabricObject} obj Object to test against
-   * @param {Object} [globalPointer] x,y object of point coordinates relative to canvas used to search per pixel target.
+   * @param {Object} [pointer] point from viewport.
    * @return {Boolean} true if point is contained within an area of given object
    * @private
    */
-  _checkTarget(
-    pointer: Point,
-    obj: FabricObject,
-    globalPointer: Point
-  ): boolean {
+  _checkTarget(obj: FabricObject, pointer: Point): boolean {
     if (
       obj &&
       obj.visible &&
       obj.evented &&
-      // http://www.geog.ubc.ca/courses/klink/gis.notes/ncgia/u32.html
-      // http://idav.ucdavis.edu/~okreylos/TAship/Spring2000/PointInPolygon.html
-      obj.containsPoint(pointer)
+      obj.containsPoint(
+        sendPointToPlane(pointer, undefined, this.viewportTransform),
```

Comment:
can we use this.restorePointerVpt(pointer) here?

We are taking the pointer in html coordinates and we are transitioning it to scene coordinates correct?

Reply:
this change and looking at possibly adding tests is the only thing i need to merge this

Reply:
I deliberately removed this usage
You removed restoreVpt so why leave this?
It is removed in the getPointer renaming pr, is it not?